app/src/test/java/com/xpay/kotlinutils/XpayUtilsTest.kt

```diff
@@ -109,22 +142,22 @@ class XpayUtilsTest {
         }
     }
 
+    // pay returns error when payment method is not available in payment options
     @Test(expected = IllegalArgumentException::class)
     fun pay_paymentMethodNotAvailable_throwsError() {
         XpayUtils.apiKey = "3uBD5mrj.3HSCm46V7xJ5yfIkPb2gBOIUFH4Ks0Ss"
         XpayUtils.communityId = "zogDmQW"
         XpayUtils.variableAmountID = 18
         XpayUtils.PaymentOptionsTotalAmounts =
             PaymentOptionsTotalAmounts(cash = 50.0, kiosk = 52.85)
-//        XpayUtils.userInfo = User("Mahmoud Aziz", "mabdelaziz@xpay.app", "+201226476026")
         XpayUtils.payUsing = PaymentMethods.CARD
         XpayUtils.request = serviceRequest
         runBlocking {
             XpayUtils.pay()
         }
     }
```

Comment:
this test (pay returns an error when the payment method is not available in payment options)
how are you testing this if you are not setting the available payment options firs?